app/views/exclusion.py

```diff
@@ -112,12 +120,25 @@ def exclusions(bid_id):
         if not exclusion_cart_form.result_text:
             exclusion_cart_form.result_text = item.title
         else:
-            exclusion_cart_form.result_text += ', '
+            exclusion_cart_form.result_text += ", "
             exclusion_cart_form.result_text += item.title
 
     return render_template(
         "exclusions.html",
         form=form,
         exclusion_cart_form=exclusion_cart_form,
-        bid_id=bid_id
+        bid_id=bid_id,
     )
+
+
+@exclusion_blueprint.route("/add_exclusions_to_bid/<int:bid_id>")
+@login_required
+def add_exclusions_to_bid(bid_id):
+    bid = Bid.query.get(bid_id)
+    selected_exclusion_item_ids = session.get("SelectedExclusionItemsDict", {})
+    for exclusion_link in bid.exclusion_links:
+        exclusion_link.delete()
```

Comment:
```
for exclusion_link in bid.exclusion_links:
        exclusion_link.delete()
```
i doubt you need to delete exclusion_link. You can edit exclusion in bidding_html if it`s necessary